admin/jqadm/themes/dashboard.js

```diff
@@ -689,6 +697,219 @@ Aimeos.Dashboard.Order = {
+
+		var domain = [];
+		for (var i = 0; i<dateRange.length; i++) {
+			domain.push(dateRange[i].toISOString().substr(5, 2));
+		}
+
+		var xScale = d3.scaleBand().range([0, width]).domain(domain).paddingInner(0.15);
+		var xAxis = d3.axisBottom().scale(xScale);
+
+		var svg = d3.select(selector)
+			.append("svg")
+			.attr("width", width + margin.left + margin.right)
+			.attr("height", height + margin.top + margin.bottom)
+			.append("g")
+			.attr("transform", "translate(" + margin.left + "," + margin.top + ")");
+
+		svg.append("g")
+			.attr("class", "x axis")
+			.attr("transform", "translate(0," + height + ")")
+			.call(xAxis);
+
+		var criteria = {"&&": [
+			{">=": {"order.cdate": firstdate.toISOString().substr(0, 10)}},
+			{"||": [
+				{"==": {"order.statuspayment": 5}},
+				{"==": {"order.statuspayment": 6}}
+			]}
+		]};
+
+		Aimeos.Dashboard.getData("order", "order.cmonth", criteria, "-order.cdate", limit).then(function(data) {
+			if( typeof data.data == "undefined" ) {
+				throw 'No data in response';
+			}
+
+			var yScale = d3.scaleLinear().range([height, 0]).domain([0, d3.max(data.data, function(d) { return +d.attributes; })]);
+			var yAxis = d3.axisLeft().scale(yScale).ticks(7);
+
+			svg.append("g")
+				.attr("class", "y axis left")
+				.call(yAxis);
+
+			var bars = svg.selectAll(".barcnt")
+				.data(data.data).enter()
+				.append("g").attr("class", "barcnt");
+
+			bars.append("rect")
+				.attr("class", "bar ordercount")
+				.attr("x", function(d) { return xScale(d.id.substr(5,2)); })
+				.attr("width", xScale.bandwidth() / 2 - 1)
+				.attr("y", function(d) { return yScale(+d.attributes); })
+				.attr("height", function(d) { return height - yScale(+d.attributes); })
+				.append("title").text(function(d){ return numFmt.format(d.attributes); });
+
+		}).done(function() {
+			$(selector).removeClass("loading");
+		});
+
+		Aimeos.Dashboard.getData("order", "order.cmonth", criteria, "-order.cdate", limit, "order.base.price", "sum").then(function(data) {
+
+			if( typeof data.data == "undefined" ) {
+				throw 'No data in response';
+			}
+
+			var yScale = d3.scaleLinear().range([height, 0]).domain([0, d3.max(data.data, function(d) { return +d.attributes; })]);
+			var yAxis = d3.axisRight().scale(yScale).ticks(7);
+
+			svg.append("g")
+				.attr("class", "y axis right")
+				.attr("transform", "translate(" + width + ",0)")
+				.call(yAxis);
+
+			var bars = svg.selectAll(".barsum")
+				.data(data.data).enter()
+				.append("g").attr("class", "barsum");
+
+			bars.append("rect")
+				.attr("class", "bar sum")
+				.attr("x", function(d) { return (xScale((d.id.substr(5,2))) + xScale.bandwidth() / 2 + 1); })
+				.attr("width", xScale.bandwidth() / 2 - 1)
+				.attr("y", function(d) { return yScale(+d.attributes); })
+				.attr("height", function(d) { return height - yScale(+d.attributes); })
+				.append("title").text(function(d) { return d.attributes.replace('.', ',') + ' €'; });
```

Comment:
You should use the `numFmt` variable to format the number according to the browsers locale. What if the currency isn't `€`?